src/test/java/com/amazon/opendistroforelasticsearch/security/alerting/BlockedIndexIntegrationTests.java

```diff
@@ -0,0 +1,325 @@
+package com.amazon.opendistroforelasticsearch.security.alerting;
+
+import com.amazon.opendistroforelasticsearch.security.support.ConfigConstants;
+import com.amazon.opendistroforelasticsearch.security.test.DynamicSecurityConfig;
+import com.amazon.opendistroforelasticsearch.security.test.SingleClusterTest;
+import com.amazon.opendistroforelasticsearch.security.test.helper.rest.RestHelper;
+import org.elasticsearch.action.admin.indices.alias.IndicesAliasesRequest;
+import org.elasticsearch.action.admin.indices.create.CreateIndexRequest;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.support.WriteRequest;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.rest.RestStatus;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BlockedIndexIntegrationTests extends SingleClusterTest {
```

Comment:
Can you also add some positive tests:
1. Check that a user/role that has permissions (ex: all_access) are allowed to do all these operations .
2. Check that a user that has ALL access but restricted on these alerting indices cannot read these indices.
3. Check that a user that has restricted permissions can search across all indices with "*" and not get permission denied, but get less data (based on index permission).  Ex. api: "\<endpoint\>/\*/_search"